World_Simulation/setplot.py

```diff
@@ -0,0 +1,250 @@
+    friction_data.read(os.path.join(plotdata.outdir, 'friction.data'))
+
+    # Load storm track
+    track = surgeplot.track_data(os.path.join(plotdata.outdir, 'fort.track'))
+
+    # Set afteraxes function
+    def surge_afteraxes(cd):
+        surgeplot.surge_afteraxes(cd, track, plot_direction=False,
+                                             kwargs={"markersize": 4})
+
+    # Color limits
+    surface_limits = [-5.0, 5.0]
+    speed_limits = [0.0, 3.0]
+    wind_limits = [0, 64]
+    pressure_limits = [935, 1013]
+    friction_bounds = [0.01, 0.04]
+    color_limits=[0,50]
+    def friction_after_axes(cd):
+        plt.title(r"Manning's $n$ Coefficient")
+
+    # ==========================================================================
+    #   Plot specifications
+    # ==========================================================================
+    regions = {
+		"World": {"xlimits": (clawdata.lower[0], clawdata.upper[0]),
+                        "ylimits": (clawdata.lower[1], clawdata.upper[1]),
+                        "figsize": (90, 30)},
+		"New Zealand": {"xlimits": (160,180),
+				"ylimits":(-50,-30),
+				"figsize": (50,50)}}
+
+
+    for (name, region_dict) in regions.items():
+
+        # Surface Figure
+        plotfigure = plotdata.new_plotfigure(name="Surface - %s" % name)
+        plotfigure.kwargs = {"figsize": region_dict['figsize']}
+        plotaxes = plotfigure.new_plotaxes()
+        plotaxes.title = "Surface"
+        plotaxes.xlimits = region_dict["xlimits"]
+        plotaxes.ylimits = region_dict["ylimits"]
+        plotaxes.afteraxes = surge_afteraxes
+
+        surgeplot.add_surface_elevation(plotaxes, bounds=surface_limits)
+        surgeplot.add_land(plotaxes)
+        plotaxes.plotitem_dict['surface'].amr_patchedges_show = [0] * 10
+        plotaxes.plotitem_dict['land'].amr_patchedges_show = [0] * 10
+
+        # Speed Figure
+        plotfigure = plotdata.new_plotfigure(name="Currents - %s" % name)
+        plotfigure.kwargs = {"figsize": region_dict['figsize']}
+        plotaxes = plotfigure.new_plotaxes()
+        plotaxes.title = "Currents"
+        plotaxes.xlimits = region_dict["xlimits"]
+        plotaxes.ylimits = region_dict["ylimits"]
+        plotaxes.afteraxes = surge_afteraxes
+
+        surgeplot.add_speed(plotaxes, bounds=speed_limits)
+        surgeplot.add_land(plotaxes,bounds=color_limits)
+        plotaxes.plotitem_dict['speed'].amr_patchedges_show = [0] * 10
+        plotaxes.plotitem_dict['land'].amr_patchedges_show = [0] * 10
+    #
+    # Friction field
+    #
+    plotfigure = plotdata.new_plotfigure(name='Friction')
+    plotfigure.show = friction_data.variable_friction and True
+
+    plotaxes = plotfigure.new_plotaxes()
+    plotaxes.xlimits = regions['World']['xlimits']
+    plotaxes.ylimits = regions['World']['ylimits']
+    plotaxes.title = "Manning's N Coefficient"
+    plotaxes.afteraxes = friction_after_axes
+    plotaxes.scaled = True
+    MappedGrid=0;
```

Comment:
This variable does not do anything and can be removed.

Reply:
Okay thank you!